aten/src/ATen/native/xnnpack/Common.h

```diff
@@ -9,6 +9,53 @@
 namespace at {
 namespace native {
 namespace xnnpack {
+
+struct Deleter final {
+  void operator()(const xnn_operator_t op) const {
+    xnn_delete_operator(op);
+  }
+};
+
+using Operator = std::unique_ptr<xnn_operator, Deleter>;
+
+struct ContextBase {
+  Operator op;
+
+  static constexpr float kMin = -std::numeric_limits<float>::infinity();
+  static constexpr float kMax = std::numeric_limits<float>::infinity();
+};
+
+struct ContextLinear final : public ContextBase {
+  int64_t output_channels;
+
+  ContextLinear() = default;
+
+  ContextLinear(Operator&& o, int64_t o_channels) {
+    op = std::move(o);
+    output_channels = o_channels;
+  }
+};
+
+struct ContextConv2D final : public ContextBase {
+  std::vector<int64_t> weight_size;
+  std::vector<int64_t> padding;
+  std::vector<int64_t> stride;
+  std::vector<int64_t> dilation;
+
+  ContextConv2D() = default;
+
+  ContextConv2D(Operator&& o, std::vector<int64_t> w_size,
+      std::vector<int64_t> padding_,
+      std::vector<int64_t> stride_,
+      std::vector<int64_t> dilation_) {
```

Comment:
Naming is off here.  Members should have underscores, and argument names should match, the full member names, minus the underscores.  Formatting seems off as well.

Reply:
Also, prefer member initialization syntax to assignment syntax.

Reply:
> Naming is off here. Members should have underscores, and argument names should match, the full member names, minus the underscores. Formatting seems off as well.

My understanding is that, only private member names trail with _. Hence the change. I can reverse it so that member name trail with _. But that was the reason.

Reply:
> Also, prefer member initialization syntax to assignment syntax.

Will do.

Reply:
Oh, I didn't think of public members.  I would probably still use the underscore just to be consistent, but I don't feel strongly about it.